doc/classes/DynamicFontData.xml

```diff
@@ -12,13 +12,13 @@
 	</methods>
 	<members>
 		<member name="antialiased" type="bool" setter="set_antialiased" getter="is_antialiased" default="true">
-			Controls whether the font should be rendered with anti-aliasing.
+			If [code]true[/code], the font is rendered with anti-aliasing. Default value: [code]true[/code].
```

Comment:
The default value is already documented automatically (see `default="true"` on previous line), so it should no longer be mentioned manually.

Reply:
There are a lot of these scattered through the docs, I suppose we should remove them all.

Reply:
I already did this half a year ago when we added support for extracting default values, there shouldn't be (m)any left: #30184.